esrc/rescueoverview.php

```diff
@@ -473,4 +483,86 @@ function displayNotes($row, $isCoord = 0, $isSARAgent = 0)
 		}
 	}
 }
+
+/**
+ * Check killboard and update rescue status if activity detected
+ * @param unknown $data
+ */
+function checkKbActivity($data) {
+
+	// create a new database connection
+	$database = new Database();
+	$kb = new KillboardActivity($database);
+	
+	$lastCheck = strtotime($kb->getLastKillboardCheck());
+	$yesterday = strtotime("-1 day");
+	
+	$isNewCheckRequired = $yesterday > $lastCheck;
+	
+	if (!$isNewCheckRequired) {		
+		return;
+	}
+	
+	$kb->updateLastKillboardCheck();
+
+	$rescue = new Rescue($database);
+	
+	foreach ($data as $row) {
+	
+		// have to wait one day because we don't have access to exact time SAR was opened
+		$since = date("Ymd0000", strtotime($row['lastcontact'] . '+1 day'));
```

Comment:
Maybe we should resolve the fact that we don't know the time the SAR was opened first. 
Consider this scenario:
SAR Opened on 2019-01-01 00:59
We add a day to become 2019-01-02 01:59
We then strip the time away from the date to make zkill API happy, which becomes 2019-01-02 00:00
SO: If he is killed within the first 23 hours of his SAR being opened we will never see it.

In the other scenario, where we dont add the day, we will create false positives every time the page is checked with no way of "clearing them" to say "I have checked this and it can be ignored."